examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -4,7 +4,34 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# MIS High level"
+    "# Tutorial 1: using a Quantum Device to solve MIS\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The high-level goal of this tutorial is to execute an instance of the classic NP-hard Maximum Independent Set (MIS) problem, using both classical and quantum methods.\n",
+    "\n",
+    "By the end of this notebook, you will know how to:\n",
+    "\n",
+    "- Setup import for standard MIS benchmarking DIMACS datasets.\n",
+    "- Setup compilation and execution of these graphs for execution on both Classical and Quantum Device (either an emulator or a physical QPU).\n",
+    "- Launch the execution and extract relevant results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dataset preparation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As in any MIS, we first need to load and prepare data. MIS can work with many types of graphs. For this tutorial, we will use the standard [DIMACS datasets](https://oeis.org/A265032/a265032.html) to benchmark the process."
```

Comment:
We probably want to explain what MIS is and why it's an important problem.

Reply:
Done.